src/applications/proxy-rewrite/partials/header.js

```diff
@@ -17,7 +19,9 @@ export default `
         <div class="usa-accordion">
           <div class="usa-banner-header">
             <div class="usa-grid usa-banner-inner">
-            <img src="/img/tiny-usa-flag.png" alt="U.S. flag">
+            <img src="${replaceWithStagingDomain(
```

Comment:
Would it make sense to create a helper that handles the inclusion of `https://www.va.gov`, so that only the relative path needs to be passed?